services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -22,6 +30,7 @@ export const DetailSection = ({
   setAlertCode: (code: string) => void;
 }) => {
   const { userProfile } = useAppContext() ?? {};
+  const [showReviewTeam, toggleReviewTeam] = useToggle(false);
```

Comment:
I learned something new! I didnt know about useToggle

Reply:
you can thank George for useToggle :) it is a custom hook he added back in the day for the Phone Number editing